charts/private-action-runner/README.md

```diff
@@ -0,0 +1,54 @@
+        helm install <RELEASE_NAME> datadog/private-action-runner -f ./config.yaml
+    ```
+* Go to the workflow connections https://app.datadoghq.com/workflow/connections
+* Create a new connection, select your private action runner and use `Service account authentication`
+* Create a new workflow and use a kubernetes action like `List pod` / `List deployment`
+
+## Going further
+* Adjust the service account permissions according to your needs, you can find more informations about kubernetes RBAC here https://kubernetes.io/docs/reference/access-authn-authz/rbac
+* Deploy several runners with different permissions / create different connections according to your needs
+* Private actions documentation https://docs.datadoghq.com/service_management/app_builder/private_actions
+
+## Values
+
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| common.image | string | `"us-east4-docker.pkg.dev/datadog-sandbox/apps-on-prem/onprem-runner:v0.0.1-alpha22"` | Current Datadog Private Action Runner image |
+| runners[0] | object | `{"config":{"actionsAllowlist":["com.datadoghq.kubernetes.core.listPod"],"appBuilder":{"port":9016},"ddBaseURL":"https://app.datadoghq.com","modes":["workflowAutomation","appBuilder"],"privateKey":"PRIVATE_KEY_FROM_CONFIG","urn":"URN_FROM_CONFIG"},"kubernetesPermissions":[{"apiGroups":[""],"resources":["pods"],"verbs":["list","get"]},{"apiGroups":["apps"],"resources":["deployments"],"verbs":["list","get"]}],"name":"default","replicas":1}` | Name of the Datadog Private Action Runner |
+| runners[0].config | object | `{"actionsAllowlist":["com.datadoghq.kubernetes.core.listPod"],"appBuilder":{"port":9016},"ddBaseURL":"https://app.datadoghq.com","modes":["workflowAutomation","appBuilder"],"privateKey":"PRIVATE_KEY_FROM_CONFIG","urn":"URN_FROM_CONFIG"}` | This is the configuration for the Datadog Private Action Runner |
+| runners[0].config.actionsAllowlist | list | `["com.datadoghq.kubernetes.core.listPod"]` | List of actions that the Datadog Private Action Runner is allowed to execute |
+| runners[0].config.appBuilder.port | int | `9016` | Required port for App Builder Mode |
+| runners[0].config.ddBaseURL | string | `"https://app.datadoghq.com"` | The base URL of the Datadog |
+| runners[0].config.modes | list | `["workflowAutomation","appBuilder"]` | Modes that the runner can run in |
+| runners[0].config.privateKey | string | `"PRIVATE_KEY_FROM_CONFIG"` | User to specify the runner's privateKey from the enrollment page |
+| runners[0].config.urn | string | `"URN_FROM_CONFIG"` | User to specify the runner's URN from the enrollment page |
+| runners[0].kubernetesPermissions | list | `[{"apiGroups":[""],"resources":["pods"],"verbs":["list","get"]},{"apiGroups":["apps"],"resources":["deployments"],"verbs":["list","get"]}]` | List of Kubernetes permissions that the Datadog Private Action Runner will have |
+| runners[0].replicas | int | `1` | Number of instances of Datadog Private Action Runner |
```

Comment:
```suggestion
| Key                                  | Description                                                                  | Type   | Default                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
|--------------------------------------|------------------------------------------------------------------------------|--------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| `common.image`                       | Current Datadog Private Action Runner image                                  | string | `"us-east4-docker.pkg.dev/datadog-sandbox/apps-on-prem/onprem-runner:v0.0.1-alpha22"`                                                                                                                                                                                                                                                                                                                                                                      |
| `runners[0]`                         | Name of the Datadog Private Action Runner                                    | object | `{"config":{"actionsAllowlist":["com.datadoghq.kubernetes.core.listPod"],"appBuilder":{"port":9016},"ddBaseURL":"https://app.datadoghq.com","modes":["workflowAutomation","appBuilder"],"privateKey":"PRIVATE_KEY_FROM_CONFIG","urn":"URN_FROM_CONFIG"},"kubernetesPermissions":[{"apiGroups":[""],"resources":["pods"],"verbs":["list","get"]},{"apiGroups":["apps"],"resources":["deployments"],"verbs":["list","get"]}],"name":"default","replicas":1}` |
| `runners[0].config`                  | Configuration for the Datadog Private Action Runner                          | object | `{"actionsAllowlist":["com.datadoghq.kubernetes.core.listPod"],"appBuilder":{"port":9016},"ddBaseURL":"https://app.datadoghq.com","modes":["workflowAutomation","appBuilder"],"privateKey":"PRIVATE_KEY_FROM_CONFIG","urn":"URN_FROM_CONFIG"}`                                                                                                                                                                                                             |
| `runners[0].config.actionsAllowlist` | List of actions that the Datadog Private Action Runner is allowed to execute | list   | `["com.datadoghq.kubernetes.core.listPod"]`                                                                                                                                                                                                                                                                                                                                                                                                                |
| `runners[0].config.appBuilder.port`  | Required port for App Builder Mode                                           | int    | `9016`                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
| `runners[0].config.ddBaseURL`        | The base URL of the Datadog app                                              | string | `"https://app.datadoghq.com"`                                                                                                                                                                                                                                                                                                                                                                                                                              |
| `runners[0].config.modes`            | Modes that the runner can run in                                             | list   | `["workflowAutomation","appBuilder"]`                                                                                                                                                                                                                                                                                                                                                                                                                      |
| `runners[0].config.privateKey`       | The runner's privateKey from the enrollment page                             | string | `"PRIVATE_KEY_FROM_CONFIG"`                                                                                                                                                                                                                                                                                                                                                                                                                                |
| `runners[0].config.urn`              | The runner's URN from the enrollment page                                    | string | `"URN_FROM_CONFIG"`                                                                                                                                                                                                                                                                                                                                                                                                                                        |
| `runners[0].kubernetesPermissions`   | List of Kubernetes permissions that the Datadog Private Action Runner has    | list   | `[{"apiGroups":[""],"resources":["pods"],"verbs":["list","get"]},{"apiGroups":["apps"],"resources":["deployments"],"verbs":["list","get"]}]`                                                                                                                                                                                                                                                                                                               |
| `runners[0].replicas`                | Number of instances of the Datadog Private Action Runner                     | int    | `1`                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
```

Reply:
Also, is that really the correct default value for "Name of the Datadog Private Action Runner"?

Reply:
the comments came through really messy, I fixed them so now they're much more readable!